packages/mesos-modules/buildinfo.json

```diff
@@ -1,9 +1,9 @@
 {
-  "requires": ["mesos"],
+  "requires": ["mesos", "boost-libs"],
     "single_source" : {
       "kind": "git",
       "git": "https://github.com/dcos/dcos-mesos-modules.git",
-      "ref": "1e264ece1a69837d0b5da5e397852424f9b4a3eb",
+      "ref": "f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a",
```

Comment:
Please update to `a8e7d15fd37d33d6497b1adad10a67b0d476e20b` to include the changes from https://github.com/dcos/dcos/pull/1000